src/OpenSheetMusicDisplay/Cursor.ts

```diff
@@ -167,4 +169,21 @@ export class Cursor {
   public get Hidden(): boolean {
     return this.hidden;
   }
+
+  public get AllVoicesUnderCursor(): VoiceEntry[] {
+    return this.VoicesUnderCursor();
+  }
+
+  public VoicesUnderCursor(instrument?: Instrument): VoiceEntry[] {
+    return this.iterator.CurrentVisibleVoiceEntries(instrument);
+  }
```

Comment:
First of all, one of these methods is unnecessary because they do the same thing.
Secondly, this method is just syntactic sugar or a simplication for this.iterator.CurrentVisibleVoiceEntries, so it's not really necessary, but it might make some code look better.

Reply:
By the way, where is one supposed to get the Instrument parameter from?

Reply:
> First of all, one of these methods is unnecessary because they do the same thing.
Secondly, this method is just syntactic sugar or a simplication for this.iterator.CurrentVisibleVoiceEntries, so it's not strictly necessary. Though it might make some code look better.

You're right. I didn't realize we have access to all that internal stuff from the outside now.

> By the way, where is one supposed to get the Instrument parameter from?

I used the `osmd.Sheet.Instruments` array in the tests.